lib/rubocop/cop/variable_force.rb

```diff
@@ -55,6 +55,27 @@ class VariableForce < Force # rubocop:disable Metrics/ClassLength
 
       SEND_TYPE = :send
 
+      handlers = {}
```

Comment:
I'm on the fence about whether the name `handlers` isn't a bit too generic, but I also don't have a strong preference about how to change it.